src/main/java/core/basesyntax/service/FileReaderImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(Path filePath) {
+        List<String> records;
+        try {
+            records = Files.readAllLines(filePath);
+            return records;
```

Comment:
```suggestion
        try {
            return Files.readAllLines(filePath);
```
Let’s  simplify it for now